lib/upperkut/strategy.rb

```diff
@@ -2,7 +2,7 @@
 require 'upperkut/item'
 
 module Upperkut
-  module Util
+  module Strategy
```

Comment:
I know that Util is not perfect, but Strategy does not represent any method implemented here.

Reply:
I was thinking about this module as a way to share common functionality to all strategies, but I agree that just `Strategy` does not represent it.

My intent was to prevent a `Util` module used just by strategies to be used in different contexts, cause it can mean a lack of os proper abstractions.

Anyway, should I revert the rename?

Reply:
I think so.
IMHO Strategy is more confusing than Util in this scenario.